### DISCUSSION TITLE
Check if episode files exist?

### DISCUSSION THREAD

**Discussion by josh-aliencode** [pymedusa/Medusa#10681] [Q&A]
I had an incident where I believe some of my episodes may have been deleted.
I'm wondering if there's a method to scan to see if the files do actually exist on my hard drive.. and possibly re-updating Medusa with the current episode status.

**Comment by nydave69** [comment]
Have you tried to re-scan files for that show?

**Reply by josh-aliencode** [reply]
What I had to do is Rescan my library
View the LOG by Show Queue (Refresh)
Pay attention to to the TVDB ID's that are missing directories.
Google the TVDB ID so I know which show it is.
Delete/Re-add the show.

If you knew an easier way, I wish You let me know.

**Comment by nydave69** [comment]
I just add the shows back to the show folder.
Then in the show on Medusa re-scan files..

**Reply by josh-aliencode** [reply]
Yeah. I don't know why i chose to delete and re-add. Just to be certain it grabbed the new stuff i guess. Because medusa doesn't update if something is removed.  it'll still show all the old filenames as if they're still there

**Reply by nydave69** [reply]
It should mark them archived if removed
